packages/nodes-base/nodes/ErrorTrigger.node.json

```diff
@@ -2,6 +2,7 @@
 	"node": "n8n-nodes-base.errorTrigger",
 	"nodeVersion": "1.0",
 	"codexVersion": "1.0",
+	"details": "In n8n, when a workflow  execution fails, it can start another workflow. This second workflow can be any arbitrary workflow on your n8n instance. Use the Error Trigger node as your Trigger in the Error workflow. ",
```

Comment:
nitpick
```suggestion
	"details": "In n8n, when a workflow  execution fails, it can start another workflow. This second workflow can be any arbitrary workflow on your n8n instance. Use the Error Trigger node as your Trigger in the Error workflow.",
```